log.go

```diff
@@ -0,0 +1,45 @@
+package authenticatedblockstore
+
+import (
+	"fmt"
+)
+
+type LogManager struct {
+	fileLimit int
+}
+
+// Exported LogManager constructor
+func NewLogManager() (*LogManager, error) {
+	var newLogMgr *LogManager
+	return newLogMgr, nil
```

Comment:
I think you can write:

return &LogManager{}, nil

Reply:
Same place holder logic here. Will remove it if there is nothing more to it.